invoice/src/main/java/org/killbill/billing/invoice/dao/InvoiceModelDao.java

```diff
@@ -58,6 +58,7 @@
     private InvoiceModelDao parentInvoice;
 
     private boolean isWrittenOff;
+    private boolean isRepaired;
```

Comment:
I would NACK it, but it looks like this is a pattern we already used for `isWrittenOff`... Quite tricky as we make a bunch of assumptions in a few places that the fields in the ModelDao match the columns.

Reply:
I think we have have this assumptions all over our code, don't we?